src/main/kotlin/no/nav/familie/ef/iverksett/infotrygd/SendFattetVedtakTilInfotrygdTask.kt

```diff
@@ -29,7 +30,9 @@ class SendFattetVedtakTilInfotrygdTask(private val infotrygdFeedClient: Infotryg
         val stønadstype = iverksett.fagsak.stønadstype
         val personIdenter = familieIntegrasjonerClient.hentIdenter(iverksett.søker.personIdent, true)
                 .map { it.personIdent }.toSet()
-        val startDato = iverksett.vedtak.tilkjentYtelse.andelerTilkjentYtelse.minOfOrNull { it.fraOgMed }
+        val tilkjentYtelse: TilkjentYtelse = iverksett.vedtak.tilkjentYtelse
+                                             ?: error("Finner ikke tilkjent ytelse for behandling med id=${iverksett.behandling.behandlingId}")
+        val startDato = tilkjentYtelse.andelerTilkjentYtelse.minOfOrNull { it.fraOgMed }
                         ?: error("Finner ikke noen andel med fraOgMed for behandling=$behandlingId")
```

Comment:
Ikke relatert denne PR men her kaster vi en feil hvis andeler er empty? Vi burde nog sende en tom liste til infotrygd i dette tilfelle tenker jeg

Reply:
Men vi sender jo ikke en liste, men en startdato... Gir det mening å sende "tom startdato" til infotrygd? 

Jeg har antatt at dette ikke er av interesse for infotrygd siden vi ikke innvilger noe som helst...